Test/Utils/ContrailNetworkManagerImpl.ps1

```diff
@@ -25,6 +27,38 @@ class ContrailNetworkManager {
             -Tenant $OpenStackConfig.Project
     }
 
+    [PSObject] GetResourceUrl([String] $Resource, [String] $Uuid) {
+        $RequestUrl = $this.ContrailUrl + "/" + $Resource
+
+        if(-not $Uuid) {
+            $RequestUrl += "s"
+        } else {
+            $RequestUrl += ("/" + $Uuid)
+        }
+
+        return $RequestUrl
+    }
+
+    [PSObject] SendRequest([String] $Method, [String] $Resource,
```

Comment:
In my opinion we can really simplify this API by providing helpers functions, mimicing Python `requests` library. Helpers would be `Get`, `Post`, etc. and they simply encapsulate `SendRequest` call with a concrete parameter. For example, instead of writing:

```
$this.API.SendRequest('Get', 'virtual-DNS', 'uuid', $request)
```

we would write:

```
$this.API.Get('virtual-DNS', 'uuid', $request)
```